saleor/payment/migrations/0043_populate_transaction_item_token.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 3.2.18 on 2023-03-20 12:52
+
+from django.apps import apps as registry
+from django.db import migrations
+from django.db.models.signals import post_migrate
+
+from ... import __version__
+from .tasks.saleor3_12 import (
+    update_transaction_token_field,
+    update_transaction_token_field_task,
+)
+
+
+def update_transaction_token_field_migration(apps, _schema_editor):
+    TransactionItem = apps.get_model("payment", "TransactionItem")
+
+    def on_migrations_complete(sender=None, **kwargs):
+        update_transaction_token_field_task.delay()
+
+    if __version__.startswith("3.12"):
+        sender = registry.get_app_config("account")
+        post_migrate.connect(on_migrations_complete, weak=False, sender=sender)
+    else:
+        update_transaction_token_field(transaction_item_class=TransactionItem)
+
```

Comment:
Shouldn't it be done in the next version? If we update it here, we'll need to run it in the next version again, as we will have instances with empty `token` value.

Reply:
Or maybe we should change the `token` definition to set the default value to fulfill new instances with `UUID` token value.

Reply:
@IKarbowiak  3.13 version  changes `token` from null=True to `False` and sets the `default`.
In 3.12, all places where the transaction is created have an additional implementation to set the token.
Assuming that the flow of upgrading from 3.11 to 3.12 for 0 downtime looks like below:
- turn off celery worker
- call migrations for 3.12
- turn on workers for 3.12
- turn off workers for 3.11
- turn on celery for 3.12

Celery will set the token for all missing fields, and all new transaction created from 3.12 will have token. 
But to be 100% sure, I also added additional migration in 3.13, to double check that we didn't have any empty token. Maybe it is not needed, but I would like to be sure all transactions are ready for change from `null=True` to `False`